code/game/machinery/doors/brigdoors.dm

```diff
@@ -146,15 +144,13 @@
 	for(var/obj/machinery/door/airlock/D in targets)
 		if(!D.density)
 			continue
-		spawn(0)
-			D.open()
+		D.open()
```

Comment:
i think these use spawn so they arent blocking, so invoke_async might work better here again

Reply:
invoke_async is still a spawn, if possible it's better to avoid it, also no this doesn't sleep as it has waitfor = false

there's few instances where async makes sense, and it is usually a deliberate choice as you don't have an alternative, it's not really something you should sprinkle around just because something sleeps